magento/pwa-studio
# [Doc] add static route tutorial

## Description

Revise and publish a tutorial for adding a new static route based on the intial content contributed by @rossmc 

## Related Issue

Closes PWA-301

## Acceptance 

Any developer

### Verification Stakeholders

Any develop

### Specification



### Verification Steps
1. Navigate to the `pwa-docs` directory
2. Run linter on new topic: `npm run lint src/tutorials/pwa-studio-fundamentals/add-a-static-route/index.md`
3. Verify no linting errors
3. Start HTML preview server: `npm run develop`
4. Verify no build failures
4. Navigate to the Tutorials section
5. Verify entry for adding static route tutorial appears on left nav
6. Click on that link and verify it loads the correct page

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
